gui/screens/messages.py

```diff
@@ -115,10 +143,102 @@ def accept_community_invite(self, community: str) -> 'CommunityScreen':
         return message.join_community()
 
 
+class CreateChatView(QObject):
+
+    def __init__(self):
+        super().__init__('mainWindow_CreateChatView')
+        self._confirm_button = Button('createChatView_confirmBtn')
+        self._cancel_button = Button('mainWindow_Cancel_StatusButton')
+        self._create_chat_contacts_list = List('createChatView_contactsList')
+
+    @property
+    @allure.step('Get contacts')
+    def contacts(self) -> typing.List[str]:
+        return self._create_chat_contacts_list.get_values('title')
+
+    @allure.step('Select contact in the list')
+    def select_contact(self, contact: str):
+        assert driver.waitFor(lambda: contact in self.contacts), f'Contact: {contact} not found in {self.contacts}'
+        self._create_chat_contacts_list.select(contact, 'title')
+
+    @allure.step('Create chat by adding contacts from contact list')
+    def create_chat(self, members):
+        for member in members[0:]:
+            time.sleep(0.2)
```

Comment:
strange time.sleep, it is soo small, is it nesessery?

Reply:
Yes, I tried without it, but the list content changes after selecting a user so, this small time.sleep needed, so the list updated (it also was like this in old framework)